drivers/mge-hid.c

```diff
@@ -689,6 +689,12 @@ static info_lkp_t pegasus_yes_no_info[] = {
 	{ 0, NULL, NULL, NULL }
 };
 
+static info_lkp_t eco_pegasus_yes_no_info[] = {
```

Comment:
pegasus was the code name of protection station ;)

Reply:
will remove pegasus from name also from function ?